src/test/java/org/folio/ApiTestSuite.java

```diff
@@ -450,5 +451,7 @@ class LedgerRolloverErrorServiceTestNested extends LedgerRolloverErrorServiceTes
   @Nested
   class LedgerRolloverProgressServiceTestNested extends LedgerRolloverProgressServiceTest {
   }
-
+  @Nested
+  class TransactionSummariesServiceTestNested extends TransactionSummariesServiceTest {
+  }
```

Comment:
can this test be added as well?
ClosedToOpenEncumbranceStrategyTestNested 
https://github.com/folio-org/mod-orders/pull/803/files#diff-b05f61cbbce5e1c6525656bf6ad408a948b09571c07aad6f244c3520945e903cR219